apps/storybook-react-native/.storybook/main.js

```diff
@@ -1,7 +1,8 @@
 // .storybook/main.js
 module.exports = {
   stories: [
-    '../../../packages/design-system-react-native/src/**/*.stories.@(js|jsx|ts|tsx)',
+    '../../../packages/design-system-react-native/src/**/!(temp-components)/*.stories.@(js|jsx|ts|tsx)',
+    '../stories/**/*.stories.@(js|jsx|ts|tsx)',
```

Comment:
Updating storybook configuration to include stories from `apps/storybook-react-native/stories` folder